react/AddToCartButton.tsx

```diff
@@ -253,14 +166,7 @@ const AddToCartButton: FC<Props & InjectedIntlProps> = ({
   )
 
   const ButtonWithLabel = (
-    <Button
-      block
-      disabled={
-        disabled || !available || mutationLoading || waitingOrderFormLoad
-      }
-      isLoading={mutationLoading || waitingOrderFormLoad}
-      onClick={handleClick}
-    >
```

Comment:
Did you talk with the store framework guys about removing the loading icon at adding an item to cart? I know we won't need to wait for the mutation to finish anymore (if I understood correctly), but I think it's a nice thing for the user to receive a feedback at clicking on the add to cart button.

Reply:
the user will still continue to have the toast feedback